Source/App.js

```diff
@@ -205,10 +205,10 @@
     //////////////////////////////////////////////////////////////////////////
 
     // Load the NYC buildings tileset
-    var city = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(3839) }));
+    var city = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({ url: Cesium.IonResource.fromAssetId(75343) }));
 
     // Adjust the tileset height so it's not floating above terrain
-    var heightOffset = -32;
+    var heightOffset = -0;
```

Comment:
I would actually completely remove line 211 through 200. We don't need to manually move the tileset anymore since this new version is clamped to Cesium World Terrain.

Reply:
I agree, my concerns are:
1) IMHO is didactical useful to learn API
2) dataset can change (as it already changed)
BTW I do not understand if you refer to remove line 211 and  change 217 (https://github.com/CesiumGS/cesium-workshop/blob/daf547a511a4e4539adcd11bb11b51ca7846c554/Source/App.js#L217) 
or
remove from 211 to 220... may you use <ctrl>+g to make code suggestion that I can follow.

Reply:
@OmarShehata any opinion about my comments?

Reply:
@luipir sorry, I mistyped. I meant remove line 211 through 220. So this entire block:

```
    var heightOffset = -0;
    city.readyPromise.then(function(tileset) {
        // Position tileset
        var boundingSphere = tileset.boundingSphere;
        var cartographic = Cesium.Cartographic.fromCartesian(boundingSphere.center);
        var surfacePosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, 0.0);
        var offsetPosition = Cesium.Cartesian3.fromRadians(cartographic.longitude, cartographic.latitude, heightOffset);
        var translation = Cesium.Cartesian3.subtract(offsetPosition, surfacePosition, new Cesium.Cartesian3());
        tileset.modelMatrix = Cesium.Matrix4.fromTranslation(translation);
    });
```

I think it would be good to err on the side of keeping these examples minimal (otherwise it might imply you need to do all this anytime you add a 3D Tileset). If someone wants to learn how to move a tileset up or down this is a better example to show this: https://sandcastle.cesium.com/index.html?src=3D%20Tiles%20Adjust%20Height.html

Reply:
ok, I'll update the PR